### PR TITLE
Allow customer service, aka 'staff' logins to use other org accounts

### PR DESCRIPTION
**Pull request by nicpottier**
This create a new user group "Customer Service".. Users that are in this group and also are in "staff" will be able to "service" an organization.

When servicing an organization the user's actions will be logged according to the login, but they will be able to interact with an org in the same way Administrators can.

I changed the org manage and flow dashboard to allow one click servicing of orgs as well.

### CODE REVIEW

File: temba/settings_common.py [hunk 1]
```diff
@@ -485,6 +486,21 @@
     "Granters": (
         'orgs.org_grant',
     ),
+    "Customer Support": (
+        'auth.user_list',
+        'auth.user_update',
+        'flows.flow_editor',
```
Comment (norkans7) [inline temba/settings_common.py]: what about messages, contacts, triggers,...

Comment (norkans7) [inline temba/settings_common.py]: what about messages, contacts, triggers,...?

Comment (nicpottier) [inline temba/settings_common.py]: The others are taken care of by setting ''org_id'' when we do a 'service'. staff users will get all the normal administrator permissions.

These are just so we can view a flow without servicing an org, for example when looking at the flow dashboard.

Comment (norkans7) [inline temba/settings_common.py]: oh right

**Comment by ericnewcomer** [comment]
looks sane

**Pull request merged**